### PR TITLE
[Data pipeline] Daily and forecast data storage

### PR DESCRIPTION
**Pull request by NoahNsimbe**
**_WHAT DOES THIS PR DO?_**

- [x] Computes and stores daily air quality data
- [x] Stores a copy of forecast data gotten from the predict API in the data warehouse.

**_WHAT ISSUES ARE RELATED TO THIS PR?_**

- Jira cards
    - []()
- GitHub issues
    - Closes #<enter GitHub issue number here>

**_HOW DO I TEST OUT THIS PR?_**

**_WHICH ENDPOINTS SHOULD BE READY FOR TESTING?:_** 

**_IS THERE ANY JENKINS CONSOLE LINK FOR CODE COVERAGE AND BUILD INFO?_**

**_ARE THERE ANY RELATED PRs?_**

### CODE REVIEW

File: src/airflow/airqo_etl_utils/daily_data_utils.py [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+from airqo_etl_utils.bigquery_api import BigQueryApi
+from airqo_etl_utils.constants import Tenant
+from airqo_etl_utils.data_validator import DataValidationUtils
+
+
+class DailyDataUtils:
+    @staticmethod
+    def average_data(data: pd.DataFrame) -> pd.DataFrame:
+
+        averaged_data = pd.DataFrame()
```
Comment (NoahNsimbe) [inline src/airflow/airqo_etl_utils/daily_data_utils.py:12]: Averaging hourly data

File: src/airflow/airqo_etl_utils/daily_data_utils.py [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+        data = DataValidationUtils.process_for_big_query(
+            dataframe=data,
+            table=table,
+            tenant=Tenant.ALL,
+        )
+
+        bigquery_api.load_data(
+            table=table,
+            dataframe=data,
+        )
```
Comment (NoahNsimbe) [inline src/airflow/airqo_etl_utils/daily_data_utils.py:106]: Storing daily device measurements

**Review by Baalmart — APPROVED** [review]
LGTM, thanks @NoahNsimbe

**Comment by Baalmart** [comment]
@NoahNsimbe , please endeavour to add the corresponding JIRA cards so that we can track/recognise this work accordingly. cc: @123MwanjeMike

**Pull request merged**